### DISCUSSION TITLE
Configuration of the amount of visible panes.

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [mucommander/mucommander#597] [Q&A] [answered]
I have not yet utilised muCommander. Because I believe that displaying multiple of the interface should be managed by what manages windows, am I able to configure muCommander to eternally present solely 1 pane?

**Comment by ahadas** [comment, marked as answer]
Not sure if that's what you're asking about but under the `View` menu you'll find the `Toggle single panel view` that would change the interface to display a 1 pane

**Reply by roke-julian-lockhart** [reply]
Understood. I am thankful.
